README.md

```diff
@@ -1196,6 +1196,39 @@ peered with.
 
 `peer_vpc` defines the identifier or name of the remote VPC.
 
+### Datasource Service Component
+```hcl-terraform
+data "aiven_service_component" "sc1" {
+    project = aiven_kafka.project1.project
+    service_name = aiven_kafka.service1.service_name
+    component = "kafka"
+    route = "dynamic"
+}
+```
+
+`project` and `service_name` define the project and service the service component
+belongs to.
+
+`component` is a service component name.
```

Comment:
This should mention something about the possible values. Like

`component` may match the name of the service (`cassandra`, `elasticsearch`, `grafana`, `influxdb`, `kafka`, `kafka_connect`, `mysql`, `pg` and `redis`), in which case the connection info of the service itself is returned. Some service types support additional service specific components like `kibana` for Elasticsearch, `kafka_connect`, `kafka_rest` and `schema_registry` for Kafka, and `pgbouncer` for PostgreSQL. Most service types also support `prometheus`.